### PR TITLE
improve ESXi build

### PR DESCRIPTION
**Pull request by alexsander-souza**
make fusefs handling more robust. review documentation

### CODE REVIEW

File: vmware-esxi/README.md [hunk 1]
```diff
@@ -2,21 +2,36 @@
...
+* qemu-block-extra
+* qemu-system-x86
+* qemu-utils
+* Packer - from upstream repository, v1.9.0 or newer
 * The VMware ESXi installation ISO must be downloaded manually. You can download it [here.](https://www.vmware.com/go/get-free-esxi)
 
 ## Requirements (to deploy the image)
 
-* [MAAS](https://maas.io) 2.5 or above, [MAAS](https://maas.io) 2.6 required for storage configuration
+* [MAAS](https://maas.io) 3.0 or above
```
Comment (brigaccess) [inline vmware-esxi/README.md:34]: I believe the previous section said 3.3 and above?

File: vmware-esxi/post.sh [hunk 2]
```diff
@@ -1,12 +1,13 @@
-#!/bin/sh
+#!/bin/sh -x
```
Comment (brigaccess) [inline vmware-esxi/post.sh:1]: Same question as with the other review: why do we need this?

**Review by brigaccess — APPROVED** [review]
+1 with a nit and a question

**Comment by alexsander-souza** [comment]
fixes #128

**Pull request merged**